modules/ftp.go

```diff
@@ -0,0 +1,189 @@
+package modules
+
+import (
+	"net"
+	"regexp"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zmap/zgrab2"
+)
+
+// FTPScanResults is the output of the scan. Identical to the original from zgrab, with the addition of TLSLog.
+type FTPScanResults struct {
+	Banner      string         `json:"banner,omitempty"`
+	AuthTLSResp string         `json:"auth_tls_resp,omitempty"`
```

Comment:
We've generally been pretty verbose in our field naming, and I suspect `auth_tls_response` might be closer to what we've used in the past. @dadrian thoughts on this one?

Reply:
This was literally pasted from https://github.com/zmap/zgrab/blob/master/ztools/ftp/log.go#L19, but if it's not being used anywhere (since ftps is off by default), expanding to "response" would definitely be better.

Reply:
<sigh>, alright. We could also likely just get rid of the `_resp` and name this `auth_tls`.

Reply:
I'd go with `auth_tls`. IIRC, Drew wrote the original implementation back in 2015, and we weren't as consistent about things back then, and he likely wasn't familiar with our naming philosophy.